kroxylicious-systemtests/pom.xml

```diff
@@ -162,6 +162,10 @@
             <groupId>info.schnatterer.moby-names-generator</groupId>
             <artifactId>moby-names-generator</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I don't see any logging changes in this PR so I'm confused by the need for this. That said I think the tests probably should depend on `slf4j-api` so I'm happy to merge it.